annotationPipeline/src/main/java/org/cbioportal/annotation/AnnotationPipeline.java

```diff
@@ -72,8 +72,7 @@ private static void help(Options gnuOptions, int exitStatus)
     private static void launchJob(String[] args, String filename, String outputFilename, String isoformOverride,
             String errorReportLocation, boolean replace, Integer postIntervalSize) throws Exception
     {
-        SpringApplication app = new SpringApplication(AnnotationPipeline.class);
-        ConfigurableApplicationContext ctx = app.run(args);
+        ConfigurableApplicationContext ctx = new SpringApplicationBuilder(AnnotationPipeline.class).web(false).run(args);
```

Comment:
There’s a spring property that you can add to disable the default web startup. I’m on mobile right now otherwise I’d share but if you don’t mind waiting for Monday I can share it then

Reply:
sure - thanks! This can def wait: I do wonder if we ever want to start up web? For the command line client it might make sense to disable completely?

Reply:
can you add this to the `application.properties.EXAMPLE` and your app properties? 

```
# required to prevent embedded spring boot web server from starting up
spring.main.web-application-type=none
```

and remove the change introduced here?

Reply:
@ao508 i think in this case it might make more sense to hardcode it. E.g. if someone used a previous `properties.EXAMPLE` file to set up their annotator and they'd try to upgrade they would get a very hard to understand error. They would then have to figure out they have to make changes to their properties file. In addition it seems to me like we always want to avoid the embedded spring boot web server, so why would we make it configurable?

Reply:
If we had some `defaults.properties` that gets overridden by `application.properties` then we could add that configuration there. That might be cleaner, but not sure if there's an easy way to set that up? I don't think `.EXAMPLE` is currently configured to work that way?

Reply:
no it isn't, it's just meant to be there as an example properties file with default values. I guess we could set the other default props to point to the public genome nexus and if `application.properties` doesn't exist then the mvn build can copy the example props file similar to how it's done in the `cbioportal` project.

Reply:
Gotcha. Yeah that would be nice. Maybe we can do that as a follow up?